### PR TITLE
Back to czmq

### PR DESCRIPTION
**Pull request by ggggggggg**
I found that when using the go zeromq implementaiton many data packets were dropped mysteriously, so I switched back to the  czmq implementation.

I also found an unrelated bug with some duplicate code in `lancero.go` that I removed.

### CODE REVIEW

**Comment by joefowler** [comment]
Fixes #192.

File: .travis.yml [hunk 1]
```diff
@@ -8,3 +8,11 @@ go:
...
 
 go_import_path: github.com/usnistgov/dastard
+
+addons:
+  apt:
+    sources:
+    - sourceline: 'deb http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-draft/xUbuntu_16.04/ ./'
+      key_url: 'http://download.opensuse.org/repositories/network:/messaging:/zeromq:/git-draft/xUbuntu_16.04/Release.key'
+    packages:
+    - libczmq-dev libzmq3-dev libsodium-dev
```
Comment (joefowler) [inline .travis.yml:18]: Add a linefeed to last line.

**Comment by joefowler** [comment]
Do you want me to investigate #194 before we merge this PR? I am starting to do that.

**Comment by ggggggggg** [comment]
I pushed a commit with the code duplication removed, I'll then re-ran autotune on horton and it works. I guess I had just removed the code duplication locally and forgot to commit it. Good catch.

**Comment by ggggggggg** [comment]
No, #194 cant be separate. It doesn't obviously break anything.

**Pull request merged**